<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewVisitWindow</class>
 <widget class="QDialog" name="NewVisitWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новый визит</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="organizationLabel">
          <property name="font">
           <font>
            <family>Century Gothic</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Название учреждения</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fullNameLabel">
          <property name="font">
           <font>
            <family>Century Gothic</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ФИО</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="currentPositionLabel">
          <property name="font">
           <font>
            <family>Century Gothic</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Должность</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="specializationLabel">
          <property name="font">
           <font>
            <family>Century Gothic</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Специальность</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="dataLabel">
          <property name="font">
           <font>
            <family>Century Gothic</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Дата</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLineEdit" name="organizationLineEdit">
          <property name="font">
           <font>
            <family>Century Gothic</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fullNameLineEdit">
          <property name="font">
           <font>
            <family>Century Gothic</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="currentPositionLineEdit">
          <property name="font">
           <font>
            <family>Century Gothic</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="specializationLineEdit">
          <property name="font">
           <font>
            <family>Century Gothic</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dataLineEdit">
          <property name="font">
           <font>
            <family>Century Gothic</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="addVisitButton">
        <property name="font">
         <font>
          <family>Century Gothic</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelVisitButton">
        <property name="font">
         <font>
          <family>Century Gothic</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
